app/src/models/layer.js

```diff
@@ -6,17 +6,6 @@ const LAYERS = [{
     subscription: true,
     datasetId: config.get('layers.umdLossGainLayerDataset'),
     layerId: config.get('layers.umdLossGainLayer')
-}, {
-    // no longer on flagship
-    name: 'imazon',
-    slug: 'imazon-alerts',
-    subscription: true
-}, {
-    name: 'terrailoss',
-    slug: 'terrai-alerts',
-    subscription: true,
-    datasetId: config.get('layers.terraILayerDataset'),
-    layerId: config.get('layers.terraILayer')
 }, {
     name: 'prodes',
```

Comment:
If there is no cron for it, I think PRODES can go to?

Reply:
Yep :+1: